Source/WebCore/Sources.txt

```diff
@@ -2279,6 +2279,8 @@ platform/graphics/transforms/TransformOperations.cpp
 platform/graphics/transforms/TransformState.cpp
 platform/graphics/transforms/TransformationMatrix.cpp
 platform/graphics/transforms/TranslateTransformOperation.cpp
+platform/libwebrtc/LibWebRTCVPXVideoDecoder.cpp
+platform/libwebrtc/LibWebRTCVPXVideoEncoder.cpp
```

Comment:
That should be in some Platform cmake file, not the generic sources list imho.

Reply:
I agree with you but there's already a bunch of libwebrtc files in Sources.txt

Reply:
It probably makes more sense to have a SourcesLibWebRTC.txt but I'm not sure how all of that hooks into the Apple XCode build.

Reply:
@philn attempting to do it in https://github.com/WebKit/WebKit/pull/6413